### PR TITLE
fix: Can not create index in BinaryVector.

### PR DESCRIPTION
**Pull request by alwayslove2013**

### CODE REVIEW

File: client/src/consts/Milvus.tsx [hunk 1]
```diff
@@ -102,11 +110,10 @@ export const m_OPTIONS = [
...
 
 export const INDEX_OPTIONS_MAP = {
-  FLOAT_POINT: Object.keys(INDEX_CONFIG).map(v => ({ label: v, value: v })),
-  BINARY: [
-    { label: 'FLAT', value: 'FLAT' },
-    { label: 'IVF_FLAT', value: 'IVF_FLAT' },
-  ],
+  // not all
+  // FLOAT_POINT: Object.keys(INDEX_CONFIG).map(v => ({ label: v, value: v })),
+  FLOAT_POINT: ['IVF_FLAT', 'IVF_PQ', 'FLAT', 'HNSW', 'ANNOY',].map(v => ({ label: v, value: v })),
```
Comment (nameczz) [inline client/src/consts/Milvus.tsx:115]: I think we can still use Object.keys(INDEX_CONFIG) filtered by BIN_xxx.
Also we can delete useless code line 114.

File: client/src/consts/Milvus.tsx [hunk 2]
```diff
@@ -19,34 +19,34 @@ export const METRIC_TYPES = [
   },
   {
     value: METRIC_TYPES_VALUES.SUBSTRUCTURE,
-    label: 'Substructure',
+    label: 'SUBSTRUCTURE',
```
Comment (Tumao727) [inline client/src/consts/Milvus.tsx:22]: Why we need to uppercase metric options label?

Comment (nameczz) [reply client/src/consts/Milvus.tsx:22]: Milvus need all uppercase value right now.so may be it's better to make label uppercase too.

**Review by nameczz — APPROVED** [review]

File: client/src/pages/schema/Create.tsx [hunk 3]
```diff
@@ -25,14 +25,15 @@ const CreateIndex = (props: {
   const { t: dialogTrans } = useTranslation('dialog');
   const { t: btnTrans } = useTranslation('btn');
 
-  const defaultMetricType = fieldType === 'BinaryVector' ? 'Hamming' : 'L2';
+  const defaultIndexType = fieldType === 'BinaryVector' ? 'BIN_IVF_FLAT': 'IVF_FLAT';
```
[the remark this answers is not in the document]
Comment (alwayslove2013) [reply client/src/pages/schema/Create.tsx]: Modified. Thanks

File: client/src/consts/Milvus.tsx (hunk 1 above)
Comment (alwayslove2013) [reply client/src/consts/Milvus.tsx]: Modified. Thanks.

**Review by shanghaikid — APPROVED** [review]

**Pull request merged**